First Maven Project

obedjamir opened the pull request:
First maven project

obedjamir committed:
First Maven Project

PinakiGhosh commented on pom.xml:
```diff
@@ -0,0 +1,22 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
```
Package name and groupId doesnot match

obedjamir replied:
i hav renamed it to com.bankbazaar.

obedjamir committed:
First Maven Project
Seperate package for each problem

obedjamir committed:
First Maven Project
Coorections made in pom.xml

PinakiGhosh commented on src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java:
```diff
@@ -0,0 +1,104 @@
+ */
+public class Card implements Comparable<Card>
+{
+    public String suite;
```
This should be enums. Not string

obedjamir replied:
Changed to enum type

PinakiGhosh commented on src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java:
```diff
@@ -0,0 +1,104 @@
+        /**
+         * Validate entry
+         */
+        if (inEnum(suite, number)==false)
```
Directly use Enum.valueOf

obedjamir replied:
valueOf used for validation

obedjamir committed:
First Maven Project
Data members of Member Class is now private.
Getters and setters used
suite and number changed to enum type in Card class
valueOf method used for card validation in Card class
